### PR TITLE
PR Feedback for test simplification

### PR DESCRIPTION
**Pull request by eshtadc**
Implementing PR feedback from https://github.com/DockYard/ember-service-worker/pull/155

### CODE REVIEW

**Comment by meirish** [comment]
Hey @eshtadc, thanks for the feedback and for implementing it! You beat me to it 😃. Sorry about the slow response.

File: lib/service-worker-builder.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ module.exports = class ServiceWorkerBuilder {
       delimiters: ['{{', '}}'],
       ROOT_URL: this.options.rootURL,
       SERVICE_WORKER_FILENAME: this.options.serviceWorkerFilename,
-      SERVICE_WORKER_SCOPE: this.options.serviceWorkerScope
+      SERVICE_WORKER_SCOPE: this.options.serviceWorkerScope || this.options.ROOT_URL
```
Comment (meirish) [inline lib/service-worker-builder.js:93]: This is a great simplification - thanks for noticing that!

**Review by meirish — APPROVED** [review]
Looks great - I’m going to merge this. Would you like me to add that other test as part of the open PR or as a folllow on? I can likely do it this weekend.

**Comment by eshtadc** [comment]
Whichever one of us gets to it. And no worries about speed - I'm happy you've stuck with it this long. We're trying to get things moving again this year after a period of really slow responses last year. I'm investigating the test failure now.

**Pull request merged**